snap/snapcraft.yaml

```diff
@@ -983,11 +983,15 @@ parts:
     plugin: cmake
     after: [ obs ]
     source: https://github.com/nowrep/obs-vkcapture.git
+    source-tag: 'v0.8.1'
```

Comment:
Any reason `0.9` or `0.9.1` wasn't used?

Reply:
Only that this was the version of `obs-vkcapture` available when the commit was made and the version that I have tested.